### PR TITLE
User Account and Family Connect Backend 

### PR DESCRIPTION
**Pull request by sydbui**
Get Live data for user and families 

## Related PRs

_Optional - any related PRs you're waiting on, or PRs that will conflict, etc_

## Screenshots

Screen Shot 2021-10-30 at 4 42 51 PM

CC: @gabriel-jt @cindyxzhang

### CODE REVIEW

File: pages/users/index.tsx [hunk 1]
```diff
@@ -2,103 +2,24 @@ import Layout from "../../components/Layout/Layout";
...
-      totalPoints: 12000,
-      familyName: "Nguyen",
-      userIds: users,
-      familyId: 12345,
-    },
-  ];
+  const [allUsers, setAllUsers] = useState([]);
+  const [allFamilies, setAllFamilies] = useState([]);
+
+  useEffect(() => {
```
Comment (gabriel-jt) [inline pages/users/index.tsx:14]: Could you look into using getServerSideProps/SSR (https://nextjs.org/docs/basic-features/data-fetching) to load the families for the users page? I have the same thoughts on scalability/response time for this page that I had with Jacob's latest PR. I have an example of this in my auth PR as well.

**Comment by gabriel-jt** [comment]
Screen Shot 2021-11-04 at 12 22 26 AM
I am not sure if this is just happening on my end again, but when I shrink the window, the spacing between the manage buttons look a bit off. Does this also happen on your end? @cindyxzhang @jacobjk01 could y'all check as well?

**Review by gabriel-jt — CHANGES_REQUESTED** [review]
Great progress Syd!! Just take a look at/address my feedback

**Review by cindyxzhang — CHANGES_REQUESTED** [review]
image

The scrolling part for the transactions is too small. Let's make the the whole modal scroll down! Can show you what I mean at worksesh

**Review by gabriel-jt — APPROVED** [review]
LGTM! awesome job syd

**Pull request merged**